README.md

```diff
@@ -106,17 +106,27 @@ Usage: mm2tei [OPTIONS] METS
 
   METS: File containing or URL pointing to the METS/MODS XML to be converted
 
+  Parse given METS and its meta-data, and convert it to TEI.
+
+  If `--ocr` is given, then also read the ALTO full-text files from the
+  fileGrp in `--text-group`, and convert page contents accordingly (in
+  physical order). Decorate page boundaries with image and page numbers, and
+  reference the corresponding base image files from `--img-group`.
+
+  Output XML to `--output (use '-' for stdout), log to stderr.`
+
 Options:
+  -O, --output FILENAME           File path to write TEI output to
   -o, --ocr                       Serialize OCR into resulting TEI
-  -T, --text-group TEXT           File group which contains the full text
+  -T, --text-group TEXT           File group which contains the full-text
```

Comment:
Sorry, `full-text` is an adjective as in `full-text search`. Either, we are so bold to assume `fulltext` or we stay with the spaced compound.

Reply:
`full-text` in `full-text search` is not an adjective – its a noun (an adjective-noun compound) modifying a noun to form a NN compound.

If you look at [this statistic](https://books.google.com/ngrams/graph?content=fulltext%2Cfull-text%2Cfull+text&year_start=1800&year_end=2019&corpus=26&smoothing=3&direct_url=t1%3B%2Cfulltext%3B%2Cc0%3B.t1%3B%2Cfull%20-%20text%3B%2Cc0%3B.t1%3B%2Cfull%20text%3B%2Cc0) you'll notice that the unhyphenated single-token form is rare. And I'll argue that the multi-word form has a different meaning, namely adjective+noun (i.e. `full` as in `complete`, `full text` as opposed to `partial text`).

Reply:
Even if you consider `full-text search` an NN compound, I still doubt that `full-text` occurs isolated as an adjective-noun compound like e.g. `The full-text has a high quality.`.